### DISCUSSION TITLE
OTA only up to version 2.08 Inovelli VZM31-SN on 1.31.2 commit: 21f5125

### DISCUSSION THREAD

**Discussion by deepcoder** [Koenkk/zigbee2mqtt#18118] [Devices]
Is there a way to check what the latest OTA firmware versions are available on Zigbee2MQTT? I just did an OTA of this VZM31-SN device on Zigbee2MQTT 1.31.2 and the update went to version 2.08. However, there are couple releases after this, as shown in link below. I see in the Zigbee2MQTT issues logs that work was done to support a 2.14 version of the Inovelli firmware. An interesting aside of what I am asked here, is that Inovelli seems to have 'pulled' the 2.14 firmware back due to some issues. That said, there are several releases listed after 2.08 and before 2.14. Thx!

https://community.inovelli.com/t/blue-series-2-1-firmware-changelog-vzm31-sn/12326

**Comment by jsenecal** [comment]
They (Inovelli) havent released the files to the public yet. https://github.com/InovelliUSA/Firmware/tree/main/Blue-Series/Zigbee/VZM31-SN-2-1-Switch
